packages/diracx-web-components/src/components/shared/Error.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Alert, AlertTitle, Button } from "@mui/material";
+import ReplayIcon from "@mui/icons-material/Replay";
+
+/**
+ * This component is used to display an error message
+ *
+ * @param s The error message (optional)
+ * @param reset The function to call to reset the error (optional)
+ * @returns Error component
+ */
+export function ErrorComponent({
+  msg,
+  reset,
+}: {
+  msg?: string;
+  reset?: () => void;
+}) {
+  return (
+    <Box
+      sx={{
+        display: "flex",
+        height: "100vh",
+        alignItems: "center",
+        justifyContent: "center",
+      }}
+      onClick={reset && reset}
```

Comment:
Is the `onClick` expected here?
```suggestion
```

Reply:
Which old component?
But you also have a button for that from what I can see. Why do you need both?